composer/core/precision.py

```diff
@@ -36,6 +36,25 @@ class Precision(StringEnum):
     AMP_FP8 = 'amp_fp8'
 
 
+def get_fp8_precision_context(
```

Comment:
can we absorb into existing precision context fn?

Reply:
👍